science/gmt5/Portfile

```diff
@@ -25,7 +28,8 @@ long_description GMT is an open source collection of ~120 tools \
     and 3D perspective views.
 
 homepage            https://www.generic-mapping-tools.org/
-master_sites        ftp://ftp.soest.hawaii.edu/gmt          \
+master_sites        https://github.com/GenericMappingTools/gmt/releases/download/${version} \
```

Comment:
Isn't this already obtained from using the GitHub portgroup? If so remove it from the list and use `master_sites-append` for the others.

Reply:
If it wasn't for the fact that this has a subport with a different version, you are right.
I tested this without the line as you suggested, but for the `gmt6` port this results in `https://github.com/GenericMappingTools/gmt/releases/download/5.4.5`, i.e., with the version specified in the main port.